src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -135,7 +135,7 @@ $3dTiles_Provider.prototype.executeCommand = function executeCommand(command) {
             pnts: this.pntsParse.bind(this),
         };
 
-        return Fetcher.arrayBuffer(url).then((result) => {
+        return Fetcher.arrayBuffer(url, layer.fetchOptions).then((result) => {
```

Comment:
I'd say +1 for `fetchOptions` vs. `textureLoaderOptions`; they are completely different, with no overlap, and different ways to say the same thing.

They need to be documented in `Core/View.js` though (`LayerOptions` type def)